app/views/layouts/application.html.erb

```diff
@@ -29,12 +29,15 @@
     <% end %>
   </head>
   <body class="govuk-template__body">
-    <div class="govuk-width-container smart_answer" id="wrapper">
-      <main id="content" role="main">
-        <%= yield %>
-      </main>
-      <%= yield :escape_link %>
-      <%= render "govuk_publishing_components/components/feedback" %>
+    <div id="wrapper">
+      <%= yield :hero %>
```

Comment:
Does the hero need to be inside the main div? The [example of MDN docs](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/main) includes the header

Reply:
The MDN example has the `<header>` outside the `<main>` element
```
<header>Gecko facts</header>

<main role="main">
  <p>Geckos are a group of usually small, usually nocturnal lizards. They are found on every continent except Australia.</p>
 
  <p>Many species of gecko have adhesive toe pads which enable them to climb walls and even windows.</p>
</main>
```

The challenge we have here with moving the hero element inside the `<main>` element is that we'll have to diverge from the Design System-based template, which constrains the main element to a container (see below) – and I would try to avoid this, especially with the [ongoing work to use this standard template across our frontend apps](https://github.com/alphagov/static/pull/2411).

```
<div class="govuk-width-container ">
  <main class="govuk-main-wrapper" id="main-content" role="main">
    <p>Geckos are a group of usually small, usually nocturnal lizards. They are found on every continent except Australia.</p>
 
    <p>Many species of gecko have adhesive toe pads which enable them to climb walls and even windows.</p>
  </main>
</div>
```